.ci/docker-compose-file/docker-compose-kafka.yaml

```diff
@@ -10,13 +10,12 @@ services:
     networks:
       emqx_bridge:
   ssl_cert_gen:
-    image: fredrikhgrelland/alpine-jdk11-openssl
+    # see https://github.com/emqx/docker-images
+    image:  ghcr.io/emqx/certgen:latest
```

Comment:
Better pin at a specific version to avoid tests breaking after a change?

Reply:
I think it's ok, we control this image and so far it's only used here.